backend/data/handlers/members.js

```diff
@@ -157,16 +142,11 @@ members.add = async (userPayload) => {
         let userSummary = _.omit(_.pick(userPayload, userSummaryFields), "_id");
         let userDetails = _.omit(_.pick(userPayload, userDetailFields), "_id");
 
-        const session = await db.startSession();
-        const response = await session.withTransaction(async () => {
-            const userDetailsResponse = await UserDetails.create(userDetails).session(session);
-            userSummary.miscDetails = userDetailsResponse._id;
-            userSummary = await replacePayloadWithIds(userSummary);
-            const res = await Member.create(userSummary).session(session);
-            return res;
-        });
-        session.endSession();
-        return response;
+        const userDetailsResponse = await UserDetails.create(userDetails);
+        userSummary.miscDetails = userDetailsResponse._id;
+        userSummary = await replacePayloadWithIds(userSummary);
+        const res = await Member.create(userSummary);
+        return res;
```

Comment:
Here, I removed many lines like: 
```
const session = await db.startSession();
```
the lines above existed so that these multiple database queries all run as part of 1 **transaction**.

I had trouble configuring my local env. to support and test MongoDB transactions, so I decided to just remove all lines related to **transactions** for now.